django/contrib/auth/base_user.py

```diff
@@ -127,6 +128,18 @@ def has_usable_password(self):
     def get_full_name(self):
         raise NotImplementedError('subclasses of AbstractBaseUser must provide a get_full_name() method')
 
+    def get_hash_value(self, timestamp):
```

Comment:
Rename to something which clearly depicts what it is supposed to return, ie like `get_session_auth_hash` below. The current name is not showing a clear intent.